### PR TITLE
Universal components: new config option docs

### PR DESCRIPTION
**Pull request by dswiecki**

### CODE REVIEW

File: docs/installation_configuration_guide/ModelConfiguration.md [hunk 1]
```diff
@@ -150,20 +150,22 @@ Important thing to remember is that Kafka server addresses/schema registry addre
...
+| schemaRegistryCacheConfig.maximumSize                                       | Low        | number   | 10000            | Maximum entries size for each caches: available schemas cache and parsed schema cache                                                                                                                                                                        |
+| lowLevelComponentsEnabled                                                   | Medium     | boolean  | false            | Add low level kafka components: 'kafka-json', 'kafka-avro', 'kafka-registry-typed-json'                                                                                                                                                                      |
+| avroAsJsonSerialization                                                     | Low        | boolean  | false            | Send payload on avro schema topic as plaintext json                                                                                                                                                                                                          |
```
Comment (mproch) [inline docs/installation_configuration_guide/ModelConfiguration.md:168]: `Send and receive` or `use`?

File: docs/installation_configuration_guide/ModelConfiguration.md [hunk 2]
```diff
@@ -150,20 +150,22 @@ Important thing to remember is that Kafka server addresses/schema registry addre
...
+| schemaRegistryCacheConfig.parsedSchemaAccessExpirationTime                  | Low        | duration | 2 hours          | How long parsed schema will be cached after first access to it                                                                                                                                                                                               |
+| schemaRegistryCacheConfig.maximumSize                                       | Low        | number   | 10000            | Maximum entries size for each caches: available schemas cache and parsed schema cache                                                                                                                                                                        |
+| lowLevelComponentsEnabled                                                   | Medium     | boolean  | false            | Add low level kafka components: 'kafka-json', 'kafka-avro', 'kafka-registry-typed-json'                                                                                                                                                                      |
```
Comment (mproch) [inline docs/installation_configuration_guide/ModelConfiguration.md:167]: `low level (deprecated)`

File: docs/installation_configuration_guide/ModelConfiguration.md (hunk 1 above)
Comment (gadomsky) [inline docs/installation_configuration_guide/ModelConfiguration.md:168]: `Send and recieve json messages conformed to Avro schema`
or
`Send and recieve json messages serialized using Avro schema` ?

**Review by gadomsky — APPROVED** [review]
LGTM

**Pull request merged**